demo/app/app.module.ts

```diff
@@ -55,6 +55,7 @@ import { TsInputModule } from '@terminus/ui/input';
 import { TsLinkModule } from '@terminus/ui/link';
 import { TsLoadingOverlayModule } from '@terminus/ui/loading-overlay';
 import { TsLoginFormModule } from '@terminus/ui/login-form';
+import { TsLogoModule } from '@terminus/ui/logo';
```

Comment:
I still get `ERROR in demo/app/app.module.ts(58,30): error TS2307: Cannot find module '@terminus/ui/logo'`here.  I think the reason is that logo component hasn't been released so by referencing it from library, it won't find the module. Once it's released, it should be resolved.

Reply:
That makes sense! Ben said he's only ever seen it in dev mode.